CMake Warning at CMakeLists.txt:54 (find_package):
  Could not find a configuration file for package "libavif" that is
  compatible with requested version "0.8.2".

  The following configuration files were considered but not accepted:

    /usr/local/lib/cmake/libavif/libavif-config.cmake, version: 1.0.0
[...]
===> Checking for items in pkg-plist which are not in STAGEDIR
Error: Missing: %%QT_PLUGINDIR%%/imageformats/kimg_avif.so
Error: Missing: share/kservices5/qimageioplugins/avif.desktop

--- CMakeLists.txt.orig	2023-08-05 12:37:26 UTC
+++ CMakeLists.txt
@@ -51,7 +51,7 @@ set_package_properties(OpenEXR PROPERTIES
     PURPOSE "Required for the QImage plugin for OpenEXR images"
 )
 
-find_package(libavif 0.8.2 CONFIG)
+find_package(libavif CONFIG)
 set_package_properties(libavif PROPERTIES
     TYPE OPTIONAL
     PURPOSE "Required for the QImage plugin for AVIF images"
